src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-aot.cpp

```diff
@@ -707,6 +706,121 @@ void ep_rt_aot_os_environment_get_utf16 (dn_vector_ptr_t *env_array)
 #endif
 }
 
+// Generate cryptographically strong random bytes.
+// Return 0 on success, -1 on failure.
+// copying code from SystemNative_GetCryptographicallySecureRandomBytes with slight modification
```

Comment:
What is that slight modification?

It would be best to move this to the [minipal](https://github.com/dotnet/runtime/tree/main/src/native/minipal) so that it is not copied around.